tests/tailor.js

```diff
@@ -240,6 +240,40 @@ describe('Tailor', () => {
                 })
                 .then(done, done);
         });
+
+        it('should render only primary fragments if response code is neither 2xx nor 5xx', done => {
+            nock('https://fragment')
+                .get('/1')
+                .reply(300, 'hello 1')
+                .get('/2')
+                .reply(401, 'hello 2')
+                .get('/3')
+                .reply(300, 'hello 3')
+                .get('/4')
+                .reply(401, 'hello 4');
+
+            mockTemplate.returns(
+                '<fragment src="https://fragment/1" primary></fragment>' +
+                    '<fragment src="https://fragment/2" primary></fragment>' +
```

Comment:
May we have several primary fragments?

Reply:
tailor supports it.
in this case, we will use the first primary fragment information